src/routes/docs/advanced/self-hosting/production/+page.markdoc

```diff
@@ -64,12 +64,11 @@ You can [change Appwrite's SMTP settings](/docs/advanced/self-hosting/email) and
 Backups are highly recommended for any production environment. Currently, there is no built-in script we provide to do this automatically. You must do the following to back up your Appwrite server data, stats, and files.
 
 1. Create a script to back up and restore the databases holding your Appwrite schemas. Note that trying to back up the database containers using a docker volume backup can result in a corrupted copy of your data. We recommend using the databases' built-in tools for this.
-2. Create a script to back up and restore your InfluxDB stats. If you don't care much about your server stats, you can skip this.
-3. Create a script to back up Appwrite storage volume. There are many online resources explaining different ways to backup a docker volume. When running on multiple servers, it is very recommended to use an attachable storage point. Some cloud providers offer integrated backups to such attachable mount like GCP, AWS, DigitalOcean, and the list continues.
-4. Create a script to back up `.env` and `docker-compose.yml`, which holds secrets and server configuration information.
+1. Create a script to back up Appwrite storage volume. There are many online resources explaining different ways to backup a docker volume. When running on multiple servers, it is very recommended to use an attachable storage point. Some cloud providers offer integrated backups to such attachable mount like GCP, AWS, DigitalOcean, and the list continues.
```

Comment:
For reviewers, markdoc renders all 1.s in order automatically.

Reply:
🚀  Personally, I think everyone should always use 1. for all numbered bullet points when writing markdown